### DISCUSSION TITLE
reference/env

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#23] [Announcements]
# reference/env

VanBlog 启动时，会读取一些环境变量以配置自身。 名称 必填 说明 默认值 --- --- --- --- VAN_BLOG_DATABASE_URL 否 mongoDB URL mongodb://mongo:27017/vanBlog?authSource=admin VANBLOGCDN_URL 否 CDN 部署的地址，在开启之前请不要设置...

https://vanblog.mereith.com/reference/env.html

**Comment by linzhifen5** [comment]
一直报错“未检测到 Vanblog 配置文件, 即将从环境变量中读取, 或采用默认配置”是为什么？

**Reply by gaomengsuanjia** [reply]
解决了吗大佬

**Reply by linzhifen5** [reply]
早解决了，但时间有点就，比较模糊。只记得是改了哪个地方就好了。
